src/main/backend.ts

```diff
@@ -154,8 +147,17 @@ export class MainBackend extends BaseBackend<ElectronOptions> implements CommonB
         (cloned as any)._scopeListeners = [];
         // if we use the crashpad minidump uploader we have to set extra whenever the scope updates
         if (this._options.useCrashpadMinidumpUploader === true) {
-          // @ts-ignore __INTERNAL_MINIDUMP is not assignable to event
-          captureEvent({ __INTERNAL_MINIDUMP: true });
+          getCurrentHub().captureEvent(
+            {
+              // @ts-ignore __INTERNAL_MINIDUMP is not assignable to event
+              __INTERNAL_MINIDUMP: true,
+            },
+            {
+              data: {
+                __sentry__: true,
+              },
```

Comment:
if an event has this in the hint, we will not invoke beforeSend.

Reply:
Do we even need the `__INTERNAL_MINIDUMP` then?

Reply:
Yes we do, this `__sentry__` is used internally if our SDK captures something itself.
The `__INTERNAL_MINIDUMP`, while being a workaround is something else.